### PR TITLE
Adding HTML 5 / JS / CSS 3 feature

### PR DESCRIPTION
**Pull request by ibuziuk**

### CODE REVIEW

File: features/html5.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+= HTML5/JS/CSS3
+:page-layout: features
+:page-product_id: jbt_core
+:page-feature_id: html5_js_css3
+:page-feature_order: 8
+:page-feature_tagline: Front-end Tooling
+:page-feature_image_url: images/features-html5-js-css3_icon_256px.png
+:page-issues_url: https://issues.jboss.org/browse/JBIDE/component/
```
Comment (maxandersen) [inline features/html5.adoc]: that doesn't seem to be the right url ?

File: features/html5.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+== HTML Preview
```
Comment (maxandersen) [inline features/html5.adoc]: we got way more than just live preview.

how about our CSS editing features ?

**Comment by ibuziuk** [comment]
@maxandersen the idea is the following: the initial PR will be created as an umbrella for front-end tooling  and other people involved in HTML5 / js will add extra content to it.
In terms of url, could you point me to the correct one? Assuming that front-end tooling is not just a single component, I don't know what url  suits best

**Comment by maxandersen** [comment]
if needs url then https://issues.jboss.org/browse/JBIDE/component/12311318 probably fits best (html/xml/js/css source editing)

**Comment by ibuziuk** [comment]
@maxandersen PR with updates - https://github.com/jbosstools/jbosstools-website/pull/377

**Pull request merged**